BraveWallet/WalletStrings.swift

```diff
@@ -1562,5 +1562,215 @@ extension Strings {
+      value: "Chain's currency name",
+      comment: "The title above the input field for users to input network currency symbol name in Custom Network Details Screen."
+    )
+    public static let customNetworkSymbolNamePlaceholder = NSLocalizedString(
+      "wallet.customNetworkSymbolNamePlaceholder",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enter currency name",
+      comment: "The placeholder for the input field for users to input network currency symbol name in Custom Network Details Screen."
+    )
+    public static let customNetworkSymbolTitle = NSLocalizedString(
+      "wallet.customNetworkSymbolTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Chain's currency symbol",
+      comment: "The title above the input field for users to input network currency symbol in Custom Network Details Screen."
+    )
+    public static let customNetworkSymbolPlaceholder = NSLocalizedString(
+      "wallet.customNetworkSymbolPlaceholder",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enter currency symbol",
+      comment: "The placeholder for the input field for users to input network currency symbol in Custom Network Details Screen."
+    )
+    public static let customNetworkCurrencyDecimalTitle = NSLocalizedString(
+      "wallet.customNetworkCurrencyDecimalTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Chain's currency decimals",
+      comment: "The title above the input field for users to input network currency decimal in Custom Network Details Screen."
+    )
+    public static let customNetworkCurrencyDecimalPlaceholder = NSLocalizedString(
+      "wallet.customNetworkCurrencyDecimalPlaceholder",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enter currency decimals",
+      comment: "The placeholder for the input field for users to input network currency decimal in Custom Network Details Screen."
+    )
+    public static let customNetworkCurrencyDecimalErrMsg = NSLocalizedString(
+      "wallet.customNetworkCurrencyDecimalErrMsg",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Invalid format—currency decimals must be a positive number.",
+      comment: "The error message for the input field when users input an invalid value for a custom network in Custom Network Details Screen."
+    )
+    public static let customNetworkRpcUrlsTitle = NSLocalizedString(
+      "wallet.customNetworkRpcUrlsTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "RPC URLs",
+      comment: "The title above the input fields for users to input network rpc urls in Custom Network Details Screen."
+    )
+    public static let customNetworkUrlsPlaceholder = NSLocalizedString(
+      "wallet.customNetworkUrlsPlaceholder",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enter URL(s)",
```

Comment:
I think we can just use `Enter URL` for the placeholder, URL(s) makes it seem like you're able to add more than one URL in the same field, but we're adding new fields

Reply:
`Enter URL(s)` is what the copy writer asked me to change it to. :p

Reply:
But that doesn't really make sense? 🤷‍♂️